### PR TITLE
57 create user management

### PR DESCRIPTION
**Pull request by BadrOutiti**

### CODE REVIEW

**Comment by jondef** [comment]
It's kind of a mess but it should run

File: .env
Comment (jaypi95) [inline .env:1]: Genius move, committing the .env. This should be in .gitignore, what happened?

File: js/clientJS/usermanagement.js
Comment (jaypi95) [inline js/clientJS/usermanagement.js:1]: There is a problem somewhere in here I think. When trying to change the username, the console throws this error and username does not get updated: 
image

**Review by jaypi95 — CHANGES_REQUESTED** [review]
Please remove the .env and fix the issue with updating the username. Great work otherwise, looks good.

File: .env
Comment (jondef) [reply .env:1]: > Genius move, committing the .env. This should be in .gitignore, what happened?

the commit should be gone when squashing I think so it should be okay

Comment (jaypi95) [reply .env:1]: The .env will still be in the repository including the password.

Comment (jaypi95) [reply .env:1]: Also the commit will be visible in the PR despite squashing

Comment (jondef) [reply .env:1]: > Also the commit will be visible in the PR despite squashing

but if we delete the env file now with a new commit, is it still gonna be visible? Cuz when squashifying the commits it's gonna disappear no?

File: js/clientJS/usermanagement.js
Comment (BadrOutiti) [reply js/clientJS/usermanagement.js:1]: This bug is also seen in the dev branche so it has nothing to do with this merge request.

I will make a new issue for this.

Comment (BadrOutiti) [reply js/clientJS/usermanagement.js:1]: Nevermind it was a bug, but i fixed it with the last commit.

File: .env
Comment (jaypi95-sc) [reply .env:1]: The history will only be squashed in the develop branched where it was merged into. When you look at the PRs, the commits are still in there.

**Review by jaypi95 — APPROVED** [review]
Nice fix with the usermanagement. I retested it and see that it works. I assume that the .env will get deleted from the computers of people who have it also committed for some reason. It doesn't matter since this .env has become obsolete now anyways. @SilvanBaach has changed all passwords already.

**Pull request merged**